pages/api/flows/index.js

```diff
@@ -0,0 +1,46 @@
+import dbConnect from "../../../lib/dbConnect";
+import Flow from "../../../models/Flow";
+import { getAllFlows } from "../../../services/flowService";
+
+export default async function handler(request, response) {
+  if (request.method === "GET") {
+    const flows = await getAllFlows();
+    return response.status(200).json(flows);
+  } else if (request.method === "POST") {
+    await dbConnect();
+
+    const postData = JSON.parse(request.body);
+    const newFlow = await Flow.create(postData);
+
+    return response
+      .status(201)
+      .json({ message: "Flow created", createdId: newFlow.id });
+  } else if (request.method === "PATCH") {
+    await dbConnect();
```

Comment:
Consider calling `dbConnect()` once at the very top of your handler function.

Reply:
Done